src/vs/workbench/contrib/notebook/browser/contrib/codeRenderer/codeRenderer.ts

```diff
@@ -96,7 +96,7 @@ export class NotebookCodeRendererContribution extends Disposable {
 }
 
 const workbenchContributionsRegistry = Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench);
-workbenchContributionsRegistry.registerWorkbenchContribution(NotebookCodeRendererContribution, LifecyclePhase.Eventually);
+workbenchContributionsRegistry.registerWorkbenchContribution(NotebookCodeRendererContribution, LifecyclePhase.Restored);
```

Comment:
We need the renderers registered as soon as editors are restored (they start rendering)

Reply:
ah so restored happens after eventually? lgtm